terraform/staging/outputs.tf

```diff
@@ -1,9 +0,0 @@
-output "cloudfront_certificate_validation" {
```

Comment:
This looks like an accident?

Reply:
@RachaelBooth sorry, forgot to write a comment on here. Chirag suggested we might not need these outputs any more (at least for staging) since the infra exists now. So I thought it would be fine to remove since we're seeing those outputs whenever we run apply. Happy to add back in if you disagree!

Reply:
I think we might want them for disaster recovery purposes (i.e. when building an env from scratch we do want them, although that would also be complicated by the ordering problem on validating the certificates). I would leave them in and consider at the point of writing up disaster recovery stuff / how to build a new env / doing cleanup at the end of the project.